src/hooks/useKanban.js

```diff
@@ -384,6 +384,24 @@ visitorId = 'visitor_' + Date.now() + '_' + Math.random().toString(36).substr(2,
     }
   };
 
+  // Adicionar nova função para atualizar o tipo de atividade
+  const updateActivityType = (columnId, cardId, activityType) => {
```

Comment:
![kody code-review](https://img.shields.io/badge/kody-code--review-brightgreen)

```javascript
const updateActivityType = async (columnId, cardId, activityType) => {
  if (!user) return;
  try {
```
The function should maintain consistency with other methods by requiring user authentication and including updatedAt timestamp.



<sub>Talk to Kody by mentioning @kody</sub>

<sub>Was this suggestion helpful? React with 👍 or 👎 to help Kody learn from this interaction.</sub><!-- kody-codereview -->&#8203;
&#8203;